independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenNEM",
+    "description": "An Open Platform for Australian Electricity Market Data",
+    "version": "1.0",
+    "contact": {
+      "name": "Paul Culmsee",
+      "url": "https://www.sevensigma.com.au",
+      "email": "paul.culmsee@sevensigma.com.au"
+    }
+  },
+  "host": "api.opennem.org.au",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/networks": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "code": {
+                    "type": "string",
+                    "description": "code"
+                  },
+                  "country": {
+                    "type": "string",
+                    "description": "country"
+                  },
+                  "label": {
+                    "type": "string",
+                    "description": "label"
+                  },
+                  "regions": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "code": {
+                          "type": "string",
+                          "description": "code"
+                        }
+                      }
+                    },
+                    "description": "regions"
+                  },
+                  "timezone": {
+                    "type": "string",
+                    "description": "timezone"
+                  },
+                  "interval_size": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "interval_size"

```

Comment:
@paulculmsee All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Docs  - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Okay. I made a note in the certifying step that this is an open source API and they dont publish the schema, so they will be terse. Cool with that?